injected/src/features/hover-summarization.js

```diff
@@ -0,0 +1,88 @@
+import ContentFeature from '../content-feature';
+
+const linkSelector = 'a[href],a';
+
+export default class HoverSummarization extends ContentFeature {
+    load() {
+        this.injectHoverSummarization();
```

Comment:
Probably don't want this here, it'll run before config. Side effects are better in init :)